app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,53 @@
+# typed: false
+
+class NotificationsController < ApplicationController
+  before_action :require_logged_in_user
+  before_action :set_page, only: [:all]
+  after_action :update_read_at, only: [:unread]
```

Comment:
Why only on `unread`? If it appears on `all`, the user can be assumed to have read it.

Reply:
I did it to match the [behavior in the replies_controller](https://github.com/lobsters/lobsters/blob/master/app/controllers/replies_controller.rb#L7). I changed it to include both all & unread actions

Reply:
I sure can't recall why I did that, sorry for the confusion.